UnitTests/UnitTests.UWP/Extensions/Test_FrameworkElementExtensions.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Uwp;
+using Microsoft.Toolkit.Uwp.UI;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Markup;
+
+namespace UnitTests.Extensions
+{
+    [TestClass]
+    public class Test_FrameworkElementExtensions : VisualUITestBase
+    {
+        [Ignore]
+        [TestCategory("FrameworkElementExtensions")]
+        [TestMethod]
+        public async Task Test_Ancestor_WeakReference()
+        {
+            // Need to sim loading the control, and not just load the XAML via XamlReader
+            await App.DispatcherQueue.EnqueueAsync(async () =>
+            {
+                var treeRoot = XamlReader.Load(
+           @"<Page
+                    xmlns=""http://schemas.microsoft.com/winfx/2006/xaml/presentation""
+                    xmlns:x=""http://schemas.microsoft.com/winfx/2006/xaml""
+                    xmlns:controls=""using:Microsoft.Toolkit.Uwp.UI.Controls""
+                    xmlns:ui=""using:Microsoft.Toolkit.Uwp.UI"">
+
+                    <Grid x:Name=""OuterGrid"" Width=""200"" Background=""Khaki"">
+                        <Button x:Name=""InnerButton"" ui:FrameworkElementExtensions.AncestorType=""Grid"" />
+                    </Grid>
+
+                </Page>") as Page;
+
+                Assert.IsNotNull(treeRoot, "Could not load XAML tree.");
+                await SetTestContentAsync(treeRoot);
+
+                // Need to simulate loading the control (and not just rely upon XamlReader.Load)
+                var frame = new Frame();
+                frame.Navigate(treeRoot.GetType());
```

Comment:
You might need to await this by doing another `await App.DispatcherQueue.EnqueueAsync()`
Perhaps?
```suggestion
                await App.DispatcherQueue.EnqueueAsync(() => frame.Navigate(treeRoot.GetType()));
```